docs/cql/cql-extensions.md

```diff
@@ -178,5 +178,5 @@ If the driver doesn't support it, `Config_error` will be sent instead.
 
 For more details, see:
 
-- Detailed [`design notes`](./per-partition-rate-limit.md)
-- Description of the [rate limit exceeded](./protocol-extensions.md#rate-limit-error) error
+- Detailed [design notes](https://github.com/scylladb/scylla/blob/master/docs/dev/per-partition-rate-limit.md)
```

Comment:
suggestion (can be in a followup PR):
define a global variable for `https://github.com/scylladb/scylla/blob/master/docs/dev/` to simplify links to thde dev docs, and make the doc more robust for path updates.

See https://www.sphinx-doc.org/en/master/usage/extensions/extlinks.html which we already use

Reply:
@tzach I'll do it in this PR after David replies to your questions below.
I assume I may need to remove "stable", but leave ".html". The redirects without ".html" were not working (or I couldn't make them work).

Reply:
There are more links like that, so I think adding the variable should be a different PR. 
I've created an issue to do that: https://github.com/scylladb/scylla/issues/11073.